CHANGE_LOG

```diff
@@ -1,14 +1,291 @@
 Version 0.54.0
```

Comment:
Needs the date (see e.g. Version 0.53.0 for format)

Reply:
done in 386c0fc